<?xml version="1.0" ?>
<launch>

  <arg name="camera_name"	default="live_camera"/>
  <arg name="planar_reference"	default="false"/>
  <arg if="$(arg planar_reference)"
       name="config"		default="plane"/>
  <arg unless="$(arg planar_reference)"
       name="config"		default="aist_camera_calibration"/>
  <arg name="ns"		default="/"/>

  <arg if="$(arg planar_reference)"
       name="calibration_client"	value="run_plane_calibration"/>
  <arg unless="$(arg planar_reference)"
       name="calibration_client"	value="run_volume_calibration"/>

  <node ns="$(arg ns)" name="$(arg calibration_client)"
	pkg="aist_camera_calibration" type="$(arg calibration_client).py"
	clear_params="true" output="screen">
    <rosparam file="$(dirname)/../config/$(arg camera_name).yaml"/>
    <rosparam unless="$(arg planar_reference)"
	      file="$(find aist_routines)/config/$(arg config)_config.yaml"/>
  </node>

</launch>
